File: CS/MultiGroupSummary/MyGridControl/MyGridHandler.cs
using DevExpress.XtraGrid.Views.Grid.Handler;
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid.Views.Grid;

namespace Q354185.MyXtraGrid {
    public class MyGridHandler : GridHandler {
        public MyGridHandler(GridView gridView)
            : base(gridView) {
        }
        protected override DevExpress.XtraGrid.Menu.GridViewFooterMenu CreateGridViewFooterMenu(GridView gridView) {
            return new MyGridFooterMenu(gridView);
        }
    }
}

File: VB/MultiGroupSummary/MyGridControl/MyGridHandler.vb
Imports DevExpress.XtraGrid.Views.Grid.Handler
Imports DevExpress.XtraGrid.Views.Grid

Namespace Q354185.MyXtraGrid

    Public Class MyGridHandler
        Inherits GridHandler

        Public Sub New(ByVal gridView As GridView)
            MyBase.New(gridView)
        End Sub

        Protected Overrides Function CreateGridViewFooterMenu(ByVal gridView As GridView) As DevExpress.XtraGrid.Menu.GridViewFooterMenu
            Return New MyGridFooterMenu(gridView)
        End Function
    End Class
End Namespace
